src/hotspot/share/runtime/continuationFreezeThaw.cpp

```diff
@@ -1698,6 +1839,7 @@ static int thaw_size(stackChunkOop chunk) {
   int size = chunk->max_thawing_size();
   size += frame::metadata_words; // For the top pc+fp in push_return_frame or top = stack_sp - frame::metadata_words in thaw_fast
   size += 2*frame::align_wiggle; // in case of alignments at the top and bottom
+  size += frame::metadata_words; // for preemption case (see possibly_adjust_frame)
```

Comment:
So this means it's OK to over-estimate the size here?

Reply:
Yes, this will be the space allocated in the stack by the vthread when thawing.